src/StripeTests/Infrastructure/ParameterBuilderTest.cs

```diff
@@ -164,40 +176,6 @@ public void SetsUrl()
             Assert.Equal("base_url", url);
         }
 
-        [Fact]
```

Comment:
this is going away but where is it caught?

Reply:
For keys, a runtime exception will be raised if the key is not castable to a string. I can maybe add a test for this, though I don't think it's necessary -- it would be testing the .NET runtime, not the library.

For values, the new encoding is able to handle any type (and properly recurse if the value is a container type). The previous test only existed because the original dictionary encoding plugin was unable to handle anything other dictionaries of string -> string.